instrumentation/opentelemetry-instrumentation-wsgi/tests/test_wsgi_middleware.py

```diff
@@ -484,6 +484,16 @@ def test_basic_wsgi_call(self):
         response = app(self.environ, self.start_response)
         self.validate_response(response, exporter)
 
+    def test_no_op_tracer_provider(self):
+        app = otel_wsgi.OpenTelemetryMiddleware(
+            simple_wsgi, tracer_provider=trace_api.NoOpTracerProvider
```

Comment:
Use trace_api.NoOpTracerProvider()
You should move to the instrument function initialized tracer_provider and not the Class object

Reply:
Fixed